api/api_storage.go

```diff
@@ -166,6 +166,8 @@ type StorageMiner interface {
 	MarketPendingDeals(ctx context.Context) (PendingDealInfo, error)                                                             //perm:write
 	MarketPublishPendingDeals(ctx context.Context) error                                                                         //perm:admin
 
+	GetEnabledSubsystems(ctx context.Context) ([]MinerSubsystem, error) //perm:read
```

Comment:
This does not follow the naming pattern for JSON-RPC methods. Should begin with a domain name (e.g. market, deals, etc.)

Reply:
There are a few exceptions for top-level operations, but I wonder if we want to put this into a `Runtime` category. Choices:

- `RuntimeGetEnabledSubystems`
- `EnabledSubsystems`

I think the second is fine as it's shorter and this is definitely one of those global cases.

Reply:
`RuntimeSubsystems` seems good to me - it is a category + name, and `Enabled` is really implicit in this context, given `runtime`.